docs/_includes/hero.html

```diff
@@ -1,4 +1,4 @@
-<section class="hero {% if page.hero_height %} {{ page.hero_height }} {% else %} is-medium {% endif %} is-bold is-primary" {% if page.hero_image %} style="background: url('{{ page.hero_image }}') no-repeat center center; background-size: cover;" {% endif %}>
+<section class="hero {% if page.hero_height %} {{ page.hero_height }} {% else %} is-medium {% endif %} is-bold is-primary" {% if page.hero_image %} style="background: url('{{ page.hero_image | prepend: site.baseurl }}') no-repeat center center; background-size: cover;" {% endif %}>
```

Comment:
This breaks the hero on the home page but fixes all other heros

Reply:
Just needs a `/` before "assets" in the image link in `index.markdown` file. I just went to do it but I can't push to this branch I guess.

Reply:
ah thanks